src/docs/claude-md-protocol.md

```diff
@@ -0,0 +1,32 @@
+# Global CLAUDE.md reference
+
+This document explains the design and sources for our global `CLAUDE.md`.
+
+## Sources and inspiration
+
+*   **Coding Agent Protocol**: We have distilled the principles from [this Gist by ctoth](https://gist.github.com/ctoth/d8e629209ff1d9748185b9830fa4e79f).
```

Comment:
Just list this a reference for inspiration 

We'll add others here in short time